hail/python/hail/methods/qc.py

```diff
@@ -1070,6 +1086,8 @@ def summarize_variants(mt: Union[MatrixTable, MatrixTable], show=True, *, handle
     - `contigs` (:obj:`Dict[str, int]`): Number of variants on each contig.
```

Comment:
Above this line, under `Notes`, it says this is a struct with "four" fields. Should be changed to "five"